### DISCUSSION TITLE
Allow Mailbox Access

### DISCUSSION THREAD

**Discussion by snupple** [modoboa/modoboa#2231] [Help]
Howdy,

Have a play around with modoboa to see if it works for me and I noticed when you create a SuperAdmin in Webamil) there is a option for "Allow mailboxes access"...  How does one actually access other users mailboxes?

**Comment by tonioo** [comment]
This option allows you to create a master user in dovecot. (https://doc.dovecot.org/configuration_manual/authentication/master_users/)
